components/desktop/partials/KnitProjectsWidgetItem.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="knit-projects-widget-item">
+    <div class="knit-projects-widget-item__top-section">
+      <img :src="projectLogoSource || 'images/teal-star.png'" alt="" class="knit-projects-widget-item__project-logo">
+      <h3 class="knit-projects-widget-item__title"> {{ title }} </h3>
+    </div>
+    <div class="knit-projects-widget-item__bottom-section">
+      <ul class="knit-projects-widget-item__list">
+        <li v-if="isNotEmpty(tags)" class="knit-projects-widget-item__list-item">
```

Comment:
You can make it a little bit more accessible. I suggest using `dl > dt, dd` tags. That's perfect for key-value pairs like in this case. The ideal way to write it is something like this:
```
<dl aria-label="Statystyki">
  <dt title="Komentarze">
    <span class="icon-youtubecomment" aria-hidden="true"></span>
    <span class="sr-only">Komentarze</span>
  </dt>
  <dd>8</dd>
  <dt title="Polubienia">
    <span class="icon-youtubeheart-empty" aria-hidden="true"></span>
    <span class="sr-only">Polubienia</span>
  </dt>
  <dd>41</dd>
</dl>
```

Reply:
Well I have to say you are completely right. Thanks for that comment :)

Reply:
Sorry I've experienced some issues while trying to implement your idea. I did use your snippet but in order to make it look right I would have to wrap dt dd in a div or something like that to display it in block. Does it make sense? If not may I use ul and apply your suggestions about aria-label and descriptions?

Reply:
Yes, you can wrap dd dt pairs in a div tag. W3C validator allows only div tag used in dl except dt and dd :)

Reply:
@zrebcu411 Thanks :)